vpr/SRC/route/route_timing.c

```diff
@@ -503,9 +538,16 @@ static int get_max_pins_per_net(void) {
 	return (max_pins_per_net);
 }
 
+struct Criticality_comp {
+	const float* criticality;
+
+	Criticality_comp(const float* calculated_criticalities) : criticality{calculated_criticalities} {}
+	bool operator()(int a, int b) const {return criticality[a] > criticality[b];}
+};
+
 bool timing_driven_route_net(int inet, int itry, float pres_fac, float max_criticality,
 		float criticality_exp, float astar_fac, float bend_cost,
-		float *pin_criticality, int *sink_order,
+		float *pin_criticality, int min_incremental_reroute_fanout,
 		t_rt_node ** rt_node_of_sink, float *net_delay, t_slack * slacks) {
 

```

Comment:
Refactored out the setting up the target pins part and timing_driven_route_sink when using the heap to route to each sink